src/main/kotlin/com/grepp/nbe1_3_team04/chat/repository/ChatJDBCRepository.kt

```diff
@@ -0,0 +1,39 @@
+package com.grepp.nbe1_3_team04.chat.repository
+
+import com.grepp.nbe1_3_team04.chat.domain.Chat
+import org.springframework.jdbc.core.BatchPreparedStatementSetter
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.stereotype.Repository
+import java.sql.PreparedStatement
+import java.sql.Timestamp
+
+@Repository
+class ChatJDBCRepository(
+    private val jdbcTemplate: JdbcTemplate
+) {
+
+    fun batchInsert(list: List<Chat>){
```

Comment:
```suggestion
    @Transactional
    fun batchInsert(list: List<Chat>){
```
`@Transactional` 은 사용 못하나요?

Reply:
넵 반영하겠습니다!